test/quicklink.spec.js

```diff
@@ -1,5 +1,8 @@
 describe('quicklink tests', function () {
   const server = `http://127.0.0.1:8080/test`;
+  // ray test touch <
+  const host = `http://127.0.0.1:8080`;
```

Comment:
It would probably make more sense to change these lines as follows:

```js
const host = `http://127.0.0.1:8080`;
const server = `${host}/test`;
```

Reply:
Thank you for teaching.
Yes, it makes way more sense.

For now, I have not updated the Unit Test accordingly.
As you think the latest updates are close enough, I'm going to update the Unit Test.

Reply:
Done